### PR TITLE
fixed auth

### PR DESCRIPTION
**Pull request by Razor0744**

### CODE REVIEW

File: app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt [hunk 1]
```diff
@@ -38,136 +34,85 @@ fun AuthScreen(
...
-                            )
-                        },
-                        textButton = stringResource(id = R.string.sign_in_user_button)
-                    )
+                LaunchedEffect(key1 = state.toastMessage) {
+                    if (state.toastMessage.isNotEmpty())
+                        Toast.makeText(activity, state.toastMessage, Toast.LENGTH_LONG).show()
                 }
+
+                SignInUserUI(
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt]: Please rename that composable function

File: app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt [hunk 2]
```diff
@@ -38,136 +34,85 @@ fun AuthScreen(
...
-                        textButton = stringResource(id = R.string.create_user_button)
-                    )
+                var isAnimationVisible by remember { mutableStateOf(false) }
+                LaunchedEffect(Unit) {
+                    isAnimationVisible = true
                 }
+
+                val coroutineScope = rememberCoroutineScope()
+
+                CreateUserUI(
```
Comment (MaximSemashko) [inline app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt]: the same

Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt]: did

File: app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt (hunk 1 above)
Comment (Razor0744) [reply app/src/main/java/com/example/movopfy/features/auth/presentation/ui/AuthScreen.kt]: did

**Pull request merged**